install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -113,6 +113,79 @@ value, triggers provisioning using the single implied
 link:#enabling-provisioner-plugins[provisioner plug-in per cloud].
 ====
 
+
+[[storage-provisioner-labels]]
+=== Storage-Provisioner Labels
+
+For dynamically provisioned storage, OpenShift defines three key/value pairs,
+called _storage-provisioner labels_ in this document,
+and associates them with the storage, in a storage-specific way.
+These are normally not visible to OpenShift users.
+Their main benefit is to enable storage administrators to recognize volumes
+associated with OpenShift.
+
+The term _storage-provisioner label_ is what this documentation uses.
+The actual term, that a storage administrator will understand, is listed
+in the following table:
+
+[cols="1,2"]
+|====
+|Term for Key/Value Pairs |Storage Type
+
+|tags
+|AWS EBS
+
+|metadata
+|OpenStack Cinder
+
+|JSON-in-description
+|GCE PD
+|====
+
+The keys describe the provisioned volume as seen by OpenShift,
+incorporating the name of the `*PersistentVolume*`
+(omitting its namespace, which is `default` and cannot be changed),
+and the namespace and name of the `*PersistentVolumeClaim*` that uses that PV.
+
+- `kubernetes.io/created-for/pv/name`
+- `kubernetes.io/created-for/pvc/namespace`
+- `kubernetes.io/created-for/pvc/name`
+
+Some example scenarios:
```

Comment:
Could this be formatted so that it's a bit more of a stand-out? Perhaps a lower-level section heading? If you go with a heading, I don't think "Some" is necessary. Just "Example Scenarios" would suffice.